plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConnectorFactory.java

```diff
@@ -50,4 +67,11 @@ public Connector create(String catalogName, Map<String, String> config, Connecto
             throw new RuntimeException(e);
         }
     }
+
+    public static class EmptyModule
```

Comment:
com.google.inject.util.Modules.EMPTY_MODULE

Reply:
i need a class, not an instance.